src/joyaxis.h

```diff
@@ -41,7 +43,7 @@ class JoyAxis : public QObject
 
   public:
     explicit JoyAxis(int index, int originset, SetJoystick *parentSet, QObject *parent);
-    ~JoyAxis();
+    virtual ~JoyAxis();
```

Comment:
Why did you change that?

Reply:
I marked this a virtual for clarity. It was already virtual before by inheriting from QObject.